junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/AbstractExtensionTest.java

```diff
@@ -23,6 +23,9 @@
 import static org.assertj.core.api.InstanceOfAssertFactories.STRING;
 
 public abstract class AbstractExtensionTest {
+    public static final String TOPIC_NAME_PATTERN = "^[a-z]+_[a-z]+$";
```

Comment:
I'm not sure the TOPIC_NAME_PATTERN constant actually helps understandability.  The pattern is used to test the value returned by the Moby Names Generator.  Calling it TOPIC_NAME_PATTERN makes it look like it used more widely.

Reply:
yeah, we've got plenty of coverage showing that the names result in topics being created. I'll drop the pattern checks.